src/ui/SmartSnippet/ExplanationModal.ts

```diff
@@ -68,26 +85,32 @@ export class ExplanationModal {
   }
 
   private buildExplanations() {
-    const explanationsContainer = $$('div', { className: EXPLANATIONS_CLASSNAME }).el;
+    const explanationsContainer = $$('fieldset', { className: EXPLANATIONS_CLASSNAME }, this.buildExplanationsLabel()).el;
     this.reasons = this.options.explanations.map(explanation => this.buildExplanationRadioButton(explanation));
     this.reasons[0].select();
     this.reasons.forEach(radioButton => explanationsContainer.appendChild(radioButton.getElement()));
     return explanationsContainer;
   }
 
+  private buildExplanationsLabel() {
```

Comment:
Could `buildReasonsLabel` be a more consistent name?

Reply:
You're right, I'll rename it along with most mentions of `explanation`.